src/components/course-listing.cjsx

```diff
@@ -5,6 +5,7 @@ Router = require 'react-router'
 WindowHelpers = require '../helpers/window'
 
 {CourseListingActions, CourseListingStore} = require '../flux/course-listing'
+{CourseStore} = require '../flux/course'
 {RefreshButton} = require 'openstax-react-components'
```

Comment:
This was addressing a different, but similar one, I wasn't aware of adding that function in when we talked about it last.